src/main.js

```diff
@@ -98,11 +98,17 @@ module.exports = {
         }
         const filePath = textEditor.getPath()
         const showRule = atom.config.get('linter-eslint.showRuleIdInMessage')
+        let rules = {}
+
+        if (textEditor.isModified()) {
+          rules = idsToIgnoredRules(atom.config.get('linter-eslint.silenceWhileTypingRules'))
```

Comment:
The rules aren't _always_ ignored though, so the name should include some indication that it is only for modified files.